### DISCUSSION TITLE
Changing resolution/fps on docker build

### DISCUSSION THREAD

**Discussion by wormvortex** [bluenviron/mediamtx#2353] [Questions]
I've followed the steps to run via docker this on my pi zero. Everything works but the resolution and fps is awful.

I'm using this:
```
docker run --rm -it \
--network=host \
--privileged \
--tmpfs /dev/shm:exec \
-v /run/udev:/run/udev:ro \
-e MTX_PATHS_CAM_SOURCE=rpiCamera \
bluenviron/mediamtx:latest-rpi
```

The guide says:
`Camera settings can be changed by using the rpiCamera* parameters:`

I do not understand this. I've found the link to the sample config and understand what I need to write but I've no idea where I find this file to edit.

**Comment by aler9** [comment]
Hello, the README explains how to change the configuration:

https://github.com/bluenviron/mediamtx#configuration

In particular, the bit rate of the camera (i.e. resolution) is controlled by the `rpiCameraBitrate` setting, that can be passed through environment variables:

```
docker run --rm -it \
--network=host \
--privileged \
--tmpfs /dev/shm:exec \
-v /run/udev:/run/udev:ro \
-e MTX_PATHS_CAM_SOURCE=rpiCamera \
-e MTX_PATHS_CAM_RPICAMERABITRATE=5000000 \
bluenviron/mediamtx:latest-rpi
```

[a comment by an automated account is not shown]
